src/app/(authenticated)/products/DesktopProducts.tsx

```diff
@@ -1,13 +1,28 @@
+'use client';
+
 import DesktopFilters from '~/components/DesktopFilters';
 import AllProductsListSection from '~/components/AllProductsListSection';
+import { useSearchParams } from 'next/navigation';
 
 export default function DesktopProducts() {
+  const searchParams = useSearchParams();
+  const search = searchParams?.get('search');
+
   return (
     <div className="hidden justify-center bg-background py-14 md:flex">
       <div className="flex">
         <DesktopFilters />
-        <div className="flex flex-col">
-          <div className="h-8 w-71"></div>
+        <div className="flex w-full flex-col md:w-210">
+          <div className="w-71 text-2xl">
+            {search ? (
+              <>
+                You searched for{' '}
+                <span className="font-bold">{`"${search}"`}</span>
```

Comment:
```suggestion
                <span className="font-bold">{`You searched for "${search}"`}</span>
```

Reply:
Oops, that weird thing was probably added by the linter when I committed, hadn't caught it thanks. I added the empty space inside the span, but not the whole phrase as I only want the search term to be bold.